### PR TITLE
[ds] Add primevue v4

### PR DESCRIPTION
**Pull request by mdartic**
* remove screenshots for histoire
* add packages versioning for i18n & designsystem (for publishing purpose & reuse in capel project)
* add w-full classes on input / password components, and remove p-fluid in forms

This PR breaks display compatibility of the nuxt playground, need another PR to fix displays. See #267

### CODE REVIEW

**Comment by mdartic** [comment]
CI is not working as it is not the aim of this PR.
The PR #264  is here to fix the CI by passing to pnpm.

**Review by Alketta — CHANGES_REQUESTED** [review]
I didn't check on primevue file (Aura),, I guess you remove the override in these files.
In fact, I wonder if I should keep them.

File: packages/designsystem/src/index.ts [hunk 1]
```diff
@@ -30,18 +30,20 @@ import LayoutBackground from './layouts/background.vue'
...
 export function setupLckDesignSystem(app: App, localI18n?: I18n): void {
   app.use(PrimeVue, {
     ripple: true,
-    unstyled: true,
-    pt: Aura,
+    theme: {
+      preset: Aura,
+    },
+    // unstyled: true,
+    // pt: Aura,
```
[the remark this answers is not in the document]
Comment (mdartic) [reply packages/designsystem/src/index.ts:46]: Indeed, I remove the local preset files.

File: packages/designsystem/package.json [hunk 2]
```diff
@@ -13,35 +13,43 @@
...
   "dependencies": {
-    "@locokit/i18n": "workspace:*",
     "@locokit/definitions": "workspace:*",
-    "@vee-validate/rules": "^4.12.7",
+    "@locokit/i18n": "workspace:*",
+    "@primevue/themes": "4.0.0-rc.3",
+    "@rushstack/eslint-patch": "^1.10.3",
+    "@typescript-eslint/parser": "^6.18.1",
+    "@vee-validate/rules": "^4.13.1",
+    "@vitejs/plugin-vue": "^5.0.5",
```
[the remark this answers is not in the document]
Comment (mdartic) [reply packages/designsystem/package.json:26]: Don't know, seems there are some deps twice in dependencies section.
I remove them.

File: packages/designsystem/src/layouts/background.story.vue [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+          </p-card>
+        </layout-background>
+      </div>
+    </Variant>
+  </Story>
+</template>
+
+<script setup lang="ts">
+import LayoutBackground from './background.vue'
+import PCard from 'primevue/card'
```
[the remark this answers is not in the document]
Comment (mdartic) [reply packages/designsystem/src/layouts/background.story.vue:51]: Right, done.

File: packages/designsystem/src/plugins/primevue.ts [hunk 4]
```diff
@@ -1,13 +1,22 @@
...
-
-import Aura from '@/presets/aura'
+import Aura from '@primevue/themes/aura'
+// import Aura from '@/presets/aura'
 
 import ConfirmationService from 'primevue/confirmationservice'
 import ToastService from 'primevue/toastservice' // theme
 import PrimeVue from 'primevue/config'
 
 export function definePluginPrime(app: App): void {
```
[the remark this answers is not in the document]
Comment (mdartic) [reply packages/designsystem/src/plugins/primevue.ts:9]: The `definePluginPrime` is only for prime whereas `setupLckDesignSystem` is larger than that, and exposed in the "root" exports of the package.
It allows the user of the `@locokit/designsystem` package to configure with only one function the design system, with prime, i18n and vee-validate.

**Review by Alketta — APPROVED** [review]

**Pull request merged**